client/src/components/related_outfits/Compare.jsx

```diff
@@ -40,12 +40,21 @@ const Compare = ({
   }
 
   return (
-    <div className="c-compare-modal" role="button" tabIndex="0" onKeyPress={() => compareClickHandler()} onClick={() => compareClickHandler()} data-testid="compare">
+    <div className="c-compare-modal" onClick={() => compareClickHandler()} data-testid="compare">
```

Comment:
removed accessibility on click handler, role="button", tabIndex, onKeyPress, etc. removed from click event? Still getting used to this but maybe meant to be included?